content/programming/in_open_source_also_communication_is_hard.rst

```diff
@@ -17,7 +17,7 @@ In open source also, communication is hard
 A large software project is above a social exercise in which technical
 experts try to reach good decisions together, for instance on github
 pull requests. But communication is difficult. It is easy to
-underestimate how much well-intended human beings can talk across
```

Comment:
I know I said this was technically correct (and I still think it is !), but it's much less common -- "talking past" is would be a little clearer in this context !